src/schema/v2/me/bidder_position_messages.ts

```diff
@@ -35,6 +36,13 @@ export const bidderPositionMessages: BiddingMessage[] = [
     getDescription: () =>
       "Sorry, your bid wasn’t received before the lot closed.",
   },
+  {
+    status: "LOT_CLOSED",
+    message: "Lot Closed to Bids",
+    header: "Lot closed",
+    getDescription: () =>
+      "Sorry, your bid wasn’t received before the lot closed.",
```

Comment:
Where is this getting used? (it looks like we tried to consolidate some of these in MP but Force doesn't use them)?

Basically wondering if you needed to make an update in Force as well, (based on your screenshot), or if Force already used this field it auto-magically would work.

Reply:
I'm p sure it's not being used anywhere but I don't think the `SALE_CLOSED` `getDescription` is being used either? I can omit here. Force work is necessary: https://github.com/artsy/force/pull/9721

Reply:
We can leave it then, it's just a bit cargo-culty! (and yes just realized the Force side PR 😅 )